Demo/src/main/java/com/braintreepayments/demo/DemoActivity.java

```diff
@@ -46,16 +50,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
     public BraintreeClient getBraintreeClient() {
```

Comment:
We should be able to remove the `getBrainreeClient()` eventually. If it can be removed in this PR that would be a nice win. Generally the less `DemoActivity` does the better.

Reply:
Good call - removed